src/ovs/ovs_control.cpp

```diff
@@ -1012,7 +1016,7 @@ void OVS_Control::monitor_vconn(vconn *vconn, bool reply_to_echo_requests,
             The pin.packet here has the same memory address, even after multiple calls.
             If you intent to store it somewhere, it is advised to make a copy of it.
           */
-                    ACA_On_Demand_Engine::get_instance().parse_packet(in_port, pin.packet);
+                    // ACA_On_Demand_Engine::get_instance().parse_packet(in_port, pin.packet);
```

Comment:
@zzxgzgz Any specific reason that the on demand engine is disabled here?

Reply:
This code path is not triggered by current logic, since the packet-in monitor is replaced by libfluid. But I agree it should be turned back on if somebody would like to use vconn.

Reply:
> @zzxgzgz Any specific reason that the on demand engine is disabled here?

This code path is not active now. The `parse_packet` will now be called in `of_controller.cpp`, inside of the `message_callback` function, when there is a `packet_in`.

Reply:
@zzxgzgz    I have uncommented this line, if somebody would like to use vconn. Please start a new merging PR of it.

Reply:
@xieus @zzxgzgz   Solved in https://github.com/futurewei-cloud/alcor-control-agent/pull/277 , please wait for the CI/CD passing and review. Thanks.